src/components/vr_cooperation/include/vr_cooperation/commands/on_default_service_chosen_notification.h

```diff
@@ -34,24 +34,25 @@
 #define SRC_COMPONENTS_VR_COOPERATION_INCLUDE_VR_COOPERATION_COMMANDS_ON_DEFAULT_SERVICE_CHOSEN_NOTIFICATION_H_
 
 #include <string>
-#include "vr_cooperation/commands/request_from_hmi.h"
-#include "vr_cooperation/vr_module.h"
+#include "vr_cooperation/commands/command.h"
 #include "vr_cooperation/interface/hmi.pb.h"
 
 namespace vr_cooperation {
 
+class VRModule;
+
 namespace commands {
 
 /**
  * @brief OnDefaultServiceChosenNotification command class
  */
-class OnDefaultServiceChosenNotification : public RequestFromHMI {
+class OnDefaultServiceChosenNotification : public Command {
  public:
   /**
    * @brief OnDefaultServiceChosenNotification class constructor
    * @param parent pointer to VRModule
    * @param message Message from HMI
-   **/
+   */
   OnDefaultServiceChosenNotification(
       VRModule* parent,
       const vr_hmi_api::ServiceMessage& message);
```

Comment:
If you use Eclipse you can use format rules from Google for Eclipse. https://github.com/google/styleguide/blob/gh-pages/eclipse-cpp-google-style.xml